### PR TITLE
Update sample reporting data

### LINKED ISSUE #1703: Feature: Add in all KRIs to `reportingResults`
## Feature Details

Add longtudinal data for KRI0003-KRI0012 and then update `reportingMetrics`, `reportingBounds` of reporting data model as needed.

### LINKED ISSUE #1793: QC: Add sample data generation scripts to `data_raw/`
## QC Details
To increase transparency and reproducibility, add scripts that generate the sample data frame CSVs

### LINKED ISSUE #1800: QC: Update sample reports to include longitudinal data
## QC Details
Currently just using one snapshot, update to use Reporting model

### PR DESCRIPTION
**Pull request by lauramaxwell**
## Overview

Update sample reporting data to: 
1. be longitudinal
2. include all KRIs
3. be available for site and country reporting
4. Include data simulation script in `data-raw` directory

This new longitudinal reporting data will be used in the sample reports in `pkgdown`, which addresses #1703.

**Note**: The site-level reporting data has no suffix, but country-level reporting data has `_country` suffix.

## Connected Issues

- Closes #1800 
- Closes #1793
- Closes #1703

### CODE REVIEW

File: .github/workflows/pkgdown-dev2main.yaml [hunk 1]
```diff
@@ -33,88 +33,40 @@ jobs:
       - name: Generate sample reports
         shell: Rscript {0}
         run: |
-          devtools::load_all()
+          devtools::load_all(dependencies = T)
```
Comment (jonthegeek) [inline .github/workflows/pkgdown-dev2main.yaml:36]: `pkgload::load_all()` (called via `devtools::load_all()` doesn't have a `dependencies` argument. This should throw an error. Were you thinking of `devtools::install()`?

File: .github/workflows/pkgdown.yaml [hunk 2]
```diff
@@ -32,90 +32,43 @@ jobs:
       - name: Generate sample reports
         shell: Rscript {0}
         run: |
-          devtools::load_all()
+          devtools::load_all(dependencies = T)
```
Comment (jonthegeek) [inline .github/workflows/pkgdown.yaml:35]:
```suggestion
          devtools::load_all()
```
See above.

**Review by jonthegeek — APPROVED** [review]
This looks great! Unfortunately... it won't work for gsm.app. I need us to export *all* of the tables (the datasets used to construct the reports), and that doesn't make sense in gsm. Can/should we make a gsm.sampledata package that owns the input data (the stuff you randomly generated here)? It's generally considered better to separate large datasets from the "main" package so you don't have to keep archiving those datasets with every release, otherwise I'd ask that we include it all in GSM.

I guess technically I could regenerate the data (since you include a seed), but it doesn't really make sense for it to be "inside" gsm.app, either.

File: .github/workflows/pkgdown-dev2main.yaml (hunk 1 above)
Comment (lauramaxwell) [reply .github/workflows/pkgdown-dev2main.yaml:36]: oh whoops- that can actually just go away- i was thinking about addressing #1804, but then stopped part-way through and forgot to get rid of all the evidence. but yeah- i was probably thinking of `install` here.

**Review by jonthegeek — APPROVED** [review]
That should do it!

**Pull request merged**